### PR TITLE
Added clients accounts to the notary account

### PR DESCRIPTION
**Pull request by Mingela**
### Description of the Change
Added saving of created user accounts to 'notary' account in a format:
`usernamedomain -> domain`

### CODE REVIEW

**Comment by Mingela** [comment]
We should probably eliminate side chain-specific writes after transition to the Iroha account creation as the only way of actual registering

File: notary-registration/src/main/kotlin/registration/NotaryRegistrationStrategy.kt [hunk 1]
```diff
@@ -13,7 +13,8 @@ import sidechain.iroha.util.ModelUtil
  */
 @Component
 class NotaryRegistrationStrategy(
-    @Autowired private val irohaConsumer: IrohaConsumer
+    @Autowired private val irohaConsumer: IrohaConsumer,
+    @Autowired private val notaryAccount: String
```
Comment (dolgopolovwork) [inline notary-registration/src/main/kotlin/registration/NotaryRegistrationStrategy.kt]: I'd rather add `@Qualifier`. `String` is a very popular class. Somebody may create another `String` bean by mistake.

File: notary-registration/src/main/kotlin/registration/NotaryRegistrationAppConfiguration.kt [hunk 2]
```diff
@@ -41,4 +42,7 @@ class NotaryRegistrationAppConfiguration {
     @Bean
     fun registrationConfig() = registrationConfig
 
+    @Bean
+    @Qualifier("notaryAccount")
```
Comment (dolgopolovwork) [inline notary-registration/src/main/kotlin/registration/NotaryRegistrationAppConfiguration.kt]: No need to do it here. Bean is called after `notaryAccount()` function name

File: configs/registration_local.properties [hunk 3]
```diff
@@ -5,6 +5,7 @@ registration.port=8085
 registration.registrationCredential.accountId=registration_service@notary
 registration.registrationCredential.pubkeyPath=deploy/iroha/keys/registration_service@notary.pub
 registration.registrationCredential.privkeyPath=deploy/iroha/keys/registration_service@notary.priv
+registration.notaryAccount=notary@notary
```
Comment (Alexey-N-Chernyshov) [inline configs/registration_local.properties]: I would name it as "registration .client_storage_account". Notary is not meaningful.

**Comment by Mingela** [comment]
Will be combined with overall brvs integration PR

**Pull request closed without merging**